src/sass/modules/_m-vet-nav.scss

```diff
@@ -35,18 +35,25 @@ body.va-pos-fixed {
     padding: 0 1.6rem;
     white-space: normal;
   }
+
+  &.vetnav--submenu-expanded {
+    #vetnav-menu {
+      overflow: hidden;
+      height: 0;
+    }
+  }
 }
```

Comment:
The submenu works by hiding the upper-level menu via setting the height to zero. This hides all of the parent menu items, while the submenu is positioned absolute so it still shows up.